### DISCUSSION TITLE
Como hacer la parte del pastelero practica 1 HTML y CSS

### DISCUSSION THREAD

**Discussion by eskaider** [Launch-X-Latam/MisionFrontEnd#243] [Q&A]
compañeros tengo una duda respecto a donde dice acerca de que el pastelero debera saber lo que pide el cliente esta se tiene que entrelazar y como hacer el inventario anexo imagen de los requisitos ya que ya tengo lo que es la web para el usuario pero no para la parte del pastelero espero puedan ayudarme o donde poder ver estudiar sobre ese inventario
![Screenshot_20221021_194721](https://user-images.githubusercontent.com/102715365/197309084-25d8457a-5802-46d6-8c92-605ab560cb3f.png)

**Comment by Cerecero** [comment]
Creo yo que podrias poner datos que tu inventes solo para mostrar la pagina del pastelero.

**Comment by MCurtis7** [comment]
En donde se encuentra esa práctica? No la encuentro

**Reply by geraodst117** [reply]
Tampoco puedo verla

**Reply by Evelo00** [reply]
hola, vayan a la ventana de html y ahi buscan la carpeta que dice practicas (Solo es visible desde pc)

**Reply by JohnnChavez** [reply]
Gracias por la información, pensé que la practica era otra

**Comment by doguedogue** [comment]
Aparte de la página de la venta de pasteles, debe haber un acceso solo para la administración de la pastelería, el menú del pastelero.

Esta parte sería para que llevaras un inventario, aunque con lo que se tiene hasta el momento no se tiene una base de datos, pero create unos datos o archivos, lo que consideres adecuado para hacer esta parte de administración que no ve el público.

**Reply by eskaider** [reply]
Suena complicado, como podría hacer eso!?

**Reply by zaazilcb** [reply]
Por lo que vi de ejemplo de una persona que subio su práctica, el habia hecho dos paginas, una de la pastelería y una del pastelero, sin embargo bastaria con añadirle una pestaña a la página prncipal que diga "administracio" o algo asi y que te lleve a la otra página para ligarlas (o eso creo). con respecto a lo que debe tener la pagina de administración, creo que pedes buscar ejemplos en internet o simplemente pensar como te gustaría ver los datos de  las ventas si tu fueras el dueño, obvio como es practica de css aun no tendria la funcionabilidad de irrestando las cantidades de producto asiq ue con que lo pongas de forma ficticia debe ser mas que suficiente
